modules/centrifuge/kreport/main.nf

```diff
@@ -12,8 +12,8 @@ process CENTRIFUGE_KREPORT {
     path db
 
     output:
-    tuple val(meta), path('*.txt')                , emit: kreport
-    path "versions.yml"                                  , emit: versions
+    tuple val(meta), path('*.txt')         , emit: kreport
```

Comment:
Better not to do this in this PR as it'll make a confusing commit message / module version description (i.e. referring to Kaiju but in the centrifuge module?)

Reply:
I agree!